<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotoSlideShow</class>
 <widget class="QWidget" name="PhotoSlideShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>671</height>
   </rect>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QGridLayout" name="frameGLayout">
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="albumLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Album Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="leftHSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="imgLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Image</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="rightHSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonHLayout">
     <item>
      <widget class="QPushButton" name="pushButton_ShowDetails">
       <property name="text">
        <string>Show/Hide Details</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttonLHSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="fullscreenButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/images/window_fullscreen.png</normaloff>:/images/window_fullscreen.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ShiftLeft">
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_StartStop">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ShiftRight">
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttonRHSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
